guides/source/association_basics.md

```diff
@@ -1184,7 +1184,7 @@ If the associated object has already been retrieved from the database for this o
 
 ##### `association=(associate)`
 
-The `association=` method assigns an associated object to this object. Behind the scenes, this means extracting the primary key from this object and setting the associated object's foreign key to the same value.
+The `association=` method assigns an associated object to this object. Behind the scenes, this means extracting the primary key from this object and setting the associated object's foreign key to the same value. No modification or deletion of existing records takes place.
```

Comment:
I think we should add it to "Methods Added by `belongs_to`" section

Reply:
Also, It might be better to add to the api docs to `belongs_to` https://github.com/rails/rails/blob/ce1248a5b8944606e91edf1bc4f1f4962a4f658d/activerecord/lib/active_record/associations.rb#L1526

Reply:
Yes, we want to add to the API docs. The guides only give an overview. Details of how things works are in the method documentation.

Reply:
Thanks for your feedback and clarification, I've updated the PR accordingly